catalog/task/create-github-release/0.1/create-github-release.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-github-release
+spec:
+  description: >-
+    Tekton task that creates a release on github.com via the GitHub API
+  params:
+    - name: repository
+      type: string
+      description: "The github repository where the release should be created"
+    - name: release_version
+      type: string
+      description: "The version string of the new release"
+    - name: githubSecret
+      type: string
+      description: "The kube secret to use to authenticate to GitHub, containing one key: token"
+    - name: content_directory
+      type: string
+      description: "The directory inside the workspace to find files for release"
+      default: "release/"
+  workspaces:
+    - name: data
+      description: The workspace where the binaries to release reside
+  results:
+    - name: url
+      type: string
+      description: URL to inspect the created release
+  steps:
+    - name: create-release-from-binaries
+      image: registry.access.redhat.com/ubi9/ubi:latest@sha256:351ed8b24d440c348486efd99587046e88bb966890a9207a5851d3a34a4dd346
+      script: |
+        #!/usr/bin/env bash
+        curl -L \
+          https://github.com/cli/cli/releases/download/v2.32.1/gh_2.32.1_linux_amd64.tar.gz \
+          --output gh_2.32.1_linux_amd64.tar.gz && \
+        tar -zvxf gh_2.32.1_linux_amd64.tar.gz && \
+        mv gh_2.32.1_linux_amd64/bin/gh /bin/gh && \
+        chmod +x /bin/gh && \
+        cd "/workspace/data/$CONTENT_DIRECTORY" && \
+        gh release create "$RELEASE_VERSION" *.zip *.json *SHA256SUMS --repo "$REPOSITORY"
+        # We will add .sig file when we have the sigining step done
+        #gh release create $RELEASE_VERSION *.zip *.json *.sig --repo $REPO
+        echo "https://github.com/$REPOSITORY/releases/tag/$RELEASE_VERSION" >>  $(results.url.path)
```

Comment:
Depending how the result will be used, having `\n` in the result string often causes issues, so I would recommend using `echo -n` here. Also, it seems there is no need to use `>>` here - I would suggest just `>`.

Updated line:
`echo -n "https://github.com/$REPOSITORY/releases/tag/$RELEASE_VERSION" >  $(results.url.path)`

Reply:
And then there's a nice button there where I can just click "commit suggestion".

Reply:
(I guess this is going to fail on all the "commits must be signed" checks; nevermind!)

Reply:
Updated in ce38ac5.